commands/command_divelist.cpp

```diff
@@ -544,7 +544,7 @@ void ImportDives::undoit()
 
 DeleteDive::DeleteDive(const QVector<struct dive*> &divesToDeleteIn)
 {
-	divesToDelete.dives = divesToDeleteIn.toStdVector();
+	divesToDelete.dives = std::vector<dive *>(divesToDeleteIn.begin(), divesToDeleteIn.end());
```

Comment:
Concerning commit message: fools seldom differ. :-D

Reply:
The new syntax is unintuitive and fragile.
But oh well. It's symmetric in both directions and apparently that's how one is supposed to deal with vectors...